Sources/View/Widget/ExpiryDateWidget.swift

```diff
@@ -33,18 +34,28 @@ class ExpiryDateWidget: TextWidget {
 
     override func setupLayout(field: HyperwalletField) {
         super.setupLayout(field: field)
-        pickerView = MonthYearPickerView(frame: CGRect(x: 0,
-                                                       y: UIScreen.main.bounds.height - 216,
-                                                       width: UIScreen.main.bounds.width,
-                                                       height: 216))
-        attachPickerView(pickerView: pickerView)
+        setupPickerView()
+        toolbar.setupToolBar(target: self, action: #selector(self.doneButtonTapped))
+        setupTextField()
     }
 
-    private func attachPickerView(pickerView: MonthYearPickerView) {
-        self.pickerView = pickerView
-        pickerView.toolBarDelegate = self
-        textField.inputView = self.pickerView
-        textField.inputAccessoryView = pickerView.toolbar
+    private func setupPickerView() {
+        pickerView = ExpiryDatePickerView(frame: .zero)
+        pickerView.accessibilityIdentifier = "ExpiryDateWidgetPickerAccessibilityIdentifier"
+    }
+
+    private func setupTextField() {
+        textField.inputView = pickerView
+        textField.inputAccessoryView = toolbar
+    }
+
+    @objc
+    private func doneButtonTapped() {
+        let month = pickerView.month
+        let year = String(describing: pickerView.year).suffix(startAt: 2)
+
+        textField.text = String(format: "%02d/%@", month, year)
```

Comment:
Suggestion: This format string `%02d/%@` could be defined on the `Localizable.strings`.

Reply:
moved to localization.